validator/sawtooth_validator/journal/consensus/poet/poet_enclave_simulator/poet_enclave_simulator.py

```diff
@@ -78,7 +78,9 @@ def initialize(cls, **kwargs):
         # Create an anti-Sybil ID that is unique for this validator
         cls._anti_sybil_id = \
             hashlib.sha256(
-                kwargs.get('NodeName', 'validator').encode()).hexdigest()
+                kwargs.get(
+                    'NodeName',
+                    datetime.datetime.now().isoformat()).encode()).hexdigest()
```

Comment:
IDK.  I think that we need to discuss this.  Do we even have the concept of a validator name/ID any more (aside from a validator's public signing key?)?